### PR TITLE
Update show plugins to show all available plugins as well as the loaded ones

### LINKED ISSUE #11051: `show plugins` should show all available plugins not only the loaded ones
## Steps to reproduce

```
msf5 > show plugins
msf5 >
msf5 > load sqlmap
[*] Sqlmap plugin loaded
[*] Successfully loaded plugin: Sqlmap
msf5 > show plugins
Plugins
=======
  Name    Description                                                    
      ----        -----------
  Sqlmap  sqlmap plugin for Metasploit                                                                                                         
msf5 >
```

### PR DESCRIPTION
**Pull request by dgarvit**
Fixes #11051

## Verification

List the steps needed to make sure this thing works

- [x] Start `msfconsole`
- [ ] `use exploit/windows/smb/ms08_067_netapi`
- [ ] ...
- [ ] **Verify** the thing does what it should
- [ ] **Verify** the thing does not do what it should not
- [ ] **Document** the thing and how it works ([Example](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/post/multi/gather/aws_keys.md))

### CODE REVIEW

**Comment by dgarvit** [comment]
The way I approached to the solution was that I found that the method already exists in another class. Apparently the method could not be called without instantiating an object of the class, so I created an object and called the `load_plugins` method. Please review and let me know if any changes need to be made :)

File: lib/msf/ui/console/command_dispatcher/modules.rb [hunk 1]
```diff
@@ -1112,7 +1112,7 @@ def show_evasion_options(mod) # :nodoc:
           def show_plugins # :nodoc:
             tbl = Table.new(
               Table::Style::Default,
-              'Header'  => 'Plugins',
+              'Header'  => 'Loaded Plugins',
```
Comment (Auxilus) [inline lib/msf/ui/console/command_dispatcher/modules.rb]: Isn't this method supposed to show all the `available` plugins?

**Comment by dgarvit** [comment]
In the pull request, yes. But this table generates loaded plugins, hence the title. I'll post a screenshot to make things clear.

File: lib/msf/ui/console/command_dispatcher/modules.rb (hunk 1 above)
Comment (dgarvit) [reply lib/msf/ui/console/command_dispatcher/modules.rb]: In the pull request, yes. But this table generates loaded plugins, hence the title. I'll post a screenshot to make things clear.

**Comment by dgarvit** [comment]
`show plugins` without loading any plugin:
image

`show plugins` after loading plugins:
image

**Comment by Auxilus** [comment]
@dgarvit gotcha 👍

**Comment by dgarvit** [comment]
Open for more suggestions! Else, lets merge this! :smiley:

**Comment by dgarvit** [comment]
@bcoles What does LGTM WFM mean?

**Comment by busterb** [comment]
I couldn't find a 'neater' way to access the 'list_plugins' method, so going to roll with it. Thanks!

**Comment by busterb** [comment]
## Release Notes

This updates the `show plugins` command to show all available plugins as well as loaded plugins.

**Pull request merged**